cryptpilot-convert.sh

```diff
@@ -443,7 +459,8 @@ disk::install_rpm_on_rootfs() {
 
     # Install all packages using yum inside the chroot
     chroot "${rootfs_mount_point}" rpmdb --rebuilddb --dbpath /var/lib/rpm
-    chroot "${rootfs_mount_point}" yum install -y "${all_packages[@]}"
+    net_proxy="HTTP_PROXY=$http_proxy HTTPS_PROXY=$http_proxy FTP_PROXY=$http_proxy RSYNC_PROXY=$http_proxy ALL_PROXY=$http_proxy NO_PROXY=$no_proxy"
+    chroot "${rootfs_mount_point}" /usr/bin/env $net_proxy yum install -y "${all_packages[@]}"
```

Comment:
shellcheck报错误了，https://github.com/openanolis/cryptpilot/actions/runs/17726719533/job/50368852299

用下面这个吧：

```suggestion
    chroot "${rootfs_mount_point}" /usr/bin/env ${http_proxy:+http_proxy=$http_proxy} ${https_proxy:+https_proxy=$https_proxy} ${ftp_proxy:+ftp_proxy=$ftp_proxy} ${rsync_proxy:+rsync_proxy=$rsync_proxy} ${all_proxy:+all_proxy=$all_proxy} ${no_proxy:+no_proxy=$no_proxy} yum install -y "${all_packages[@]}"
```